pandas/core/groupby/groupby.py

```diff
@@ -3772,18 +3638,15 @@ def blk_func(values: ArrayLike) -> ArrayLike:
         mgr = self._get_data_to_aggregate()
         orig_mgr_len = len(mgr)
 
-        if numeric_only_bool:
+        if numeric_only:
             mgr = mgr.get_numeric_data()
 
-        res_mgr = mgr.grouped_reduce(blk_func, ignore_failures=False)
+        res_mgr = mgr.grouped_reduce(blk_func)
 
         if not is_ser and len(res_mgr.items) != orig_mgr_len:
```

Comment:
im pretty sure L3780-L3790 should be unnecesary now

Reply:
OK for follow-up.  lets merge this sucker

Reply:
It will never end, will it? :laughing: I'll put up a followup shortly.

Reply:
ive got an "assorted cleanups" branch, will do it there